web/src/js/components/Dashboard/VideoEngagementComponent.js

```diff
@@ -8,6 +8,7 @@ import CloseIcon from '@material-ui/icons/Close'
 import Modal from '@material-ui/core/Modal'
 import Typography from '@material-ui/core/Typography'
 import useTrueX, { CLOSED } from 'js/utils/useTrueX'
+import { disabledColor } from 'js/theme/default'
```

Comment:
In general: rather than importing a value from the theme, use the theme in styles construction below. In this case for "legacy" code, feel free to leave it as-is.